tao/tao_bridge/kernels/tao_compilation_cache.cc

```diff
@@ -1156,33 +1248,44 @@ Status TaoCompilationCache::CompileImpl(
             << SignatureDebugString(signature);
 
     entry->compiled = true;
-    // Do the actual JIT compilation without holding the lock (it can take
-    // a long time.)
-    entry->compilation_status =
-        PrepareCompilerInput(function, constant_args, fixed_shape_args,
-                             host_args, variable_args, ctx, &input, is_mlir);
-    TF_RETURN_IF_ERROR(entry->compilation_status);
 
+    bool hit_cache = false;
     std::string output_file_name;
-    if (!tensorflow::Env::Default()->LocalTempFilename(&output_file_name)) {
-      return errors::Internal(
-          "couldn't get temp tao_compiler_result file name");
+    if (!disc_cache_dump_path_.empty()) {
```

Comment:
nit: if `disc_cache_dump_path_` is also used for loading, how about just call it `disc_cache_path_` ?

Reply:
done